#the header of the message
Header header

float64 brake
float64 dir
float64 speed
int32 priority
float64 lifetime